### PR TITLE
EVO-4410 Added to RabbitMQ coreId with users authenticated id

### PR DESCRIPTION
**Pull request by jacobchristensen79**

### CODE REVIEW

**Comment by hairmare** [comment]
@jacobchristensen79 This looks ok to me.

Could you please add a test that verifies that it actually works somewhere in/near `Graviton\RabbitMqBundle\Tests\Controller\EventStatusControllerTest`.

File: src/Graviton/RabbitMqBundle/Listener/EventStatusLinkResponseListener.php [hunk 1]
```diff
@@ -11,12 +11,15 @@
 use Graviton\RabbitMqBundle\Document\QueueEvent;
 use Graviton\RestBundle\HttpFoundation\LinkHeader;
 use Graviton\RestBundle\HttpFoundation\LinkHeaderItem;
+use Graviton\SecurityBundle\Authentication\Strategies\CookieFieldStrategy;
+use Graviton\SecurityBundle\Entities\SecurityUser;
```
Comment (wegenmic) [inline src/Graviton/RabbitMqBundle/Listener/EventStatusLinkResponseListener.php]: Graviton\SecurityBundle\Entities\SecurityUser not in use

File: src/Graviton/RabbitMqBundle/Listener/EventStatusLinkResponseListener.php [hunk 2]
```diff
@@ -11,12 +11,15 @@
...
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\HttpKernel\Event\FilterResponseEvent;
 use OldSound\RabbitMqBundle\RabbitMq\ProducerInterface;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\Routing\RouterInterface;
+use Symfony\Component\Security\Core\Authentication\Token\PreAuthenticatedToken;
```
Comment (wegenmic) [inline src/Graviton/RabbitMqBundle/Listener/EventStatusLinkResponseListener.php]: Graviton\SecurityBundle\Entities\SecurityUser not in use

**Comment by wegenmic** [comment]
Looks good from my side (tested the roundtrip with workers)

**Pull request merged**